### ISSUE TITLE
Firefox (Quantum), RPC with hidden element throws error Couldn't start bundle with id mapfull. Error was: TypeError: e is null

### ISSUE THREAD

**Issue by aalho** [oskariorg/oskari-docs#81]
Hi!

Having a bit of trouble here.
Using the latest version of RPC-client.
Oskari version I am communicating with seems to be 1.48.0

I have a UI where the iframe is hidden on page load (bootstrap modal windows, for example). IE, Edge and Chrome works just fine with this, but Firefox Quantum (60.2.2esr) crashes

Attached is a screenshot from the exception:
![image](https://user-images.githubusercontent.com/17899257/48410193-95197280-e746-11e8-887a-c6d093d632dc.png)

Step to reproduce: (code below):
```html
<html>
  <head>
  </head>
  <body>
    <div style="display: none">
      <iframe id="oskari" src="UrlHere"></iframe>
    </div>
  </body>
  <script src="rpc-client.min.js"></script>
  <script>
    var oskariDomain = "OSKARI_DOMAIN";
    var oskariObject = document.getElementById("oskari");
    var channel = OskariRPC.connect(oskariObject, oskariDomain); //crashes in firefox
      channel.onReady(function () {
        alert("oskari ready");
      });		
  </script>
</html>`

**Comment by ZakarFin** [comment]
Thanks for reporting this! Have to test this out, but looks like this is related to the map component startup before anything RPC-related is run.

**Comment by Miselius** [comment]
The provided code works in Firefox Quantum 62.0.3 (64-bit).

**Comment by ZakarFin** [comment]
Not really sure if there's anything that we can or should do about this if it's a browser issue that has been fixed on a more recent Firefox version.

**Comment by aalho** [comment]
Hi!

I can confirm that this issue is with the browser. Did some testing and this is resolved in Firefox version 62.0.0, so I think that it is pointless to try to fix this issue on the Oskari application.

Now I have to fight the IT-department... What is the point of ESR version anyways...

**Comment by ZakarFin** [comment]
Ok, thanks for confirming!
